components/Card.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+const Card: React.FC<Props> = ({ btncolor, heading, paragraph, onClick }) => {
+  return (
+    <div
+      style={{
+        border: "2px solid black",
+        width: "250px",
+        height: "260px",
+        boxShadow: "#837d7d 0px 0px 10px 0px",
```

Comment:
you can create css file for these stylings

Reply:
Updated